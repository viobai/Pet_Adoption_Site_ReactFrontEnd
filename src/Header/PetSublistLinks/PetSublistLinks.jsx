import React from 'react';
import { Link } from 'react-router-dom';
import style from "./PetSublistLinks.module.css";

const PetSublistLinks = () => {

    return (
        <div className={style.dropContainer}>
            <Link className={style.categoryLink} to="/petlist">All</Link>
            <Link className={style.categoryLink} to="/petlist/dog">Dogs</Link>
            <Link className={style.categoryLink} to="/petlist/cat">Cats</Link>
            <Link className={style.categoryLink} to="/petlist/smallfurry">Small & Furry</Link>
            <Link className={style.categoryLink} to="/petlist/bird">Birds</Link>
            <Link className={style.categoryLink} to="/petlist/reptile">Reptiles</Link>
        </div>
            
    )
}
export default PetSublistLinks