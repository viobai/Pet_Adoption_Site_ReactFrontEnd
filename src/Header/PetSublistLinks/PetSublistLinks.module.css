.dropContainer {
    position: absolute;
    background-color: #f1f1f1;
    flex-direction: column;
    z-index: 2;
    margin-top: 1rem;
}
  
.categoryLink {
    color: rgb(185, 46, 28);
    padding: 12px 12px;
    text-decoration: none;
    display: block;
    text-align: center;
}
  
.categoryLink:hover {background-color: #ddd;}
