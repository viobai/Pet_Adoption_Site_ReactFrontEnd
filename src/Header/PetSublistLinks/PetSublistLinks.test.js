import React from "react";
import PetSublistLinks from "./PetSublistLinks";

describe("PetSublistLinks tests", () => {
    it("renders correctly", () => {  
        const { container } = render(<PetSublistLinks/>);
        expect(container).toMatchSnapshot();
    });
});