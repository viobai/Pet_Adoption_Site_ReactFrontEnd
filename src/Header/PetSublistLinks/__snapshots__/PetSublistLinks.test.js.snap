// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PetSublistLinks tests renders correctly 1`] = `
<div>
  <div
    class="dropContainer"
  >
    <a
      class="categoryLink"
      href="/petlist"
    >
      All
    </a>
    <a
      class="categoryLink"
      href="/petlist/dog"
    >
      Dogs
    </a>
    <a
      class="categoryLink"
      href="/petlist/cat"
    >
      Cats
    </a>
    <a
      class="categoryLink"
      href="/petlist/smallfurry"
    >
      Small & Furry
    </a>
    <a
      class="categoryLink"
      href="/petlist/bird"
    >
      Birds
    </a>
    <a
      class="categoryLink"
      href="/petlist/reptile"
    >
      Reptiles
    </a>
  </div>
</div>
`;
